cni/test/install_cni.go

```diff
@@ -305,29 +240,55 @@ func doTest(t *testing.T, chainedCNIPlugin bool, wd, preConfFile, resultFileName
 ) {
 	t.Logf("prior cni-conf='%v', expected result='%v'", preConfFile, resultFileName)
 
+	// disable monitoring & repair
+	viper.Set(constants.MonitoringPort, 0)
+	viper.Set(constants.RepairEnabled, false)
+
 	// Don't set the CNI conf file env var if preConfFile is not set
```

Comment:
```suggestion
```

Reply:
Yeah it's still set but not an env var anymore.